keyboards/evyd13/atom47/keymaps/PHSC138/keymap.c

```diff
@@ -0,0 +1,389 @@
+/* Copyright 2022 PHSC138
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "config.h"
+
+// These are all aliases for the function layers.
+#define _BASE 0
+#define _PROG 1
+#define _GAME 2
+#define _FN   7
+#define _FN1  8
+#define _NUM  9
```

Comment:
Enums are much simpler to handle, especially the skip from 2 to 7 still populates those inbetween layers, and can increase the finished compile size.
```suggestion
enum custom_layers {
    _BASE,
    _PROG,
    _GAME,
    _FN,
    _FN1,
    _NUM
};
```

Reply:
Resolved here [5c04134](https://github.com/qmk/qmk_firmware/pull/18768/commits/5c041345e76837e83a89a2d732937392c0d1e8e2)